tests/runner.html

```diff
@@ -42,6 +42,48 @@
     window.suites.push('async-loader-with-modules.html');
   } catch (e) {} // eslint-disable-line no-empty
 </script>
+<script>
+  // Include `async-loader-no-polyfills` test only for browsers that require no polyfills.
+  // Feature-detection tests are from `webcomponents-loader.js`.
```

Comment:
It's unfortunate we are copy-pasting code here, but I can't think of a better solution right now, other than adding another server-roundtrip. That is not something we want to do, so will keep this for now.

Reply:
Yup, it's not ideal.. 😞 not a fan of copy-pasting the feature detection tests from the loader too - especially since it should (by right) be kept in sync in future. But the feature detection code is quite standard (and tested) by now, so hopefully there shouldn't be too many changes to it..